diff --git setup.py setup.py
index 0dca873..eeed554 100644
--- setup.py
+++ setup.py
@@ -2,7 +2,7 @@ import os
 import sys
 from distutils.core import Extension, setup
 from Cython.Distutils import build_ext
-
+import numpy
 platform_supported = False
 for prefix in ['darwin', 'linux', 'bsd']:
     if prefix in sys.platform:
@@ -10,6 +10,7 @@ for prefix in ['darwin', 'linux', 'bsd']:
         include_dirs = [
             '/usr/include',
             '/usr/local/include',
+            numpy.get_include(),
         ]
         lib_dirs = [
             '/usr/lib',
